orchagent/aclorch.cpp

```diff
@@ -912,18 +915,31 @@ bool AclTable::create()
     attr.value.booldata = true;
     table_attrs.push_back(attr);
 
+    if(stage == ACL_STAGE_INGRESS)
+    {
+        attr.id = SAI_ACL_TABLE_ATTR_ACL_STAGE;
+        attr.value.s32 = SAI_ACL_STAGE_INGRESS;
```

Comment:
>attr.value.s32 [](start = 8, length = 14)

You could write a smaller scope if-else only for s32 field. Others are the same.

Reply:
done